### PR TITLE
migrate to go module

### PR DESCRIPTION
**Pull request by honnix**

### CODE REVIEW

**Comment by kumare3** [comment]
Some tests are failing? But this looks awesome.

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4 // indirect
+	github.com/prometheus/common v0.7.0 // indirect
+	github.com/prometheus/procfs v0.0.5 // indirect
+	github.com/satori/uuid v1.2.0 // indirect
+	github.com/sirupsen/logrus v1.4.2
+	github.com/spf13/afero v1.2.2 // indirect
+	github.com/spf13/cobra v0.0.5
+	github.com/spf13/jwalterweatherman v1.1.0 // indirect
+	github.com/spf13/pflag v1.0.5
+	github.com/spf13/viper v1.3.2
```
Comment (EngHabu) [inline go.mod]: Can you double check this includes the SHA from Gopkg.toml?

**Comment by honnix** [comment]
This and flyteidl don't work and there was weird issue that I haven't solved.

File: go.mod (hunk 1 above)
Comment (honnix) [reply go.mod]: I think this commit is included since v1.3.0
https://github.com/spf13/viper/commit/ae103

**Comment by honnix** [comment]
https://github.com/golang/go/commit/f85125345cc5d3eb054c90bfca4bda3544d7fcab might be related.

**Comment by honnix** [comment]
I believe the problem is here https://github.com/lyft/flytestdlib/blob/master/cli/pflags/api/generator.go#L304 This doesn't support module. I'm looking into using `packages` instead https://github.com/golang/go/issues/26433

**Comment by honnix** [comment]
Well it works!

**Comment by honnix** [comment]
@EngHabu PTAL again. We will need to release a new version of `pflags` cli.

**Comment by honnix** [comment]
Another thing to mention is, some test cases won't pass on Mac due to "too many opened files" in some temp folder.

**Comment by EngHabu** [comment]
Awesome!! Love it! Thank you for investigating and fixing the goimports issue
Yeah as soon as we merge this to master, please follow these steps: https://github.com/lyft/flytestdlib#releases to package and release a new version of pflags

**Pull request merged**